gradle.properties

```diff
@@ -20,3 +20,6 @@ android.useAndroidX=true
 
 GROUP_ID=com.linkedin.shaky
 VERSION_NAME=3.0.7-SNAPSHOT
+android.defaults.buildfeatures.buildconfig=true
+android.nonTransitiveRClass=false
+android.nonFinalResIds=false
```

Comment:
What's the reason for these? In a single module project non-transitive r shouldn't be relevant